src/components/Header/Header.tsx

```diff
@@ -0,0 +1,67 @@
+import styles from './Header.module.css';
+import type { LayoutText } from 'custom-types';
+import devfileLogo from '@public/images/devfileLogo.svg';
+import _layoutText from '@config/layout-text.json';
```

Comment:
Before we use the configs, do we want to have prebuild validation like how you have in registry-viewer? 🤔

Reply:
I must've forgot to add the prebuild validation, Ill add it.